### DISCUSSION TITLE
build on CPU only without GPU?

### DISCUSSION THREAD

**Discussion by silvacarl2** [ggml-org/whisper.cpp#367] [General]
i have tried to build on a CPU only without GPU, but i get a core dump when i run it:

./main -m models/ggml-base.en.bin -f samples/jfk.wav
whisper_model_load: loading model from 'models/ggml-base.en.bin'
whisper_model_load: n_vocab       = 51864
whisper_model_load: n_audio_ctx   = 1500
whisper_model_load: n_audio_state = 512
whisper_model_load: n_audio_head  = 8
whisper_model_load: n_audio_layer = 6
whisper_model_load: n_text_ctx    = 448
whisper_model_load: n_text_state  = 512
whisper_model_load: n_text_head   = 8
whisper_model_load: n_text_layer  = 6
whisper_model_load: n_mels        = 80
whisper_model_load: f16           = 1
whisper_model_load: type          = 2
whisper_model_load: adding 1607 extra tokens
whisper_model_load: mem_required  =  506.00 MB
whisper_model_load: ggml ctx size =  140.60 MB
Illegal instruction (core dumped)

just to test it, it is being built on a Hewlett-Packard HP Compaq Pro 6300 SFF, 32.0 GiB RAM, Intel® Core™ i7-3770 CPU @ 3.40GHz × 8, Ubuntu 22.04.1 LTS

any ideas what we should try?

**Comment by ggerganov** [comment]
Please provide output of the following:

```
make clean
make
```

**Comment by silvacarl2** [comment]
that nailed it.  needed to do make clean, then make.  will save for each CPU we are testing with.
